modules/js/index.js

```diff
@@ -3,7 +3,12 @@ const inputAuthor = document.querySelector('.input-author');
 const bookShelf = document.querySelector('.book-shelf');
 const bookCard = document.createElement('div');
 const bookForm = document.querySelector('.book-input');
-
+const navlist = document.querySelector('#show-list-button');
+const navadd = document.querySelector('#add-new-button');
+const navcon = document.querySelector('#contact-button');
+const list = document.querySelector('#list');
+const addNew = document.querySelector('#add');
+const connav = document.querySelector('#contact');
```

Comment:
## GOOD JOB your app is working well already 👍

- [ ] Let's divide the js files into reasonable modules and put them into the `modules` folder. And also `css` and `images` shouldn't be inside this folder 💡

Reply:
Yes. I will work on it soon